regression/Makefile

```diff
@@ -1,10 +1,12 @@
 
 DIRS = ansi-c \
        cbmc \
+      cbmc-typedef \
```

Comment:
The indent seems to be a bit off, unless that's just how github is printing this.